Formula/docx2txt.rb

```diff
@@ -0,0 +1,22 @@
+class Docx2txt < Formula
+  desc "Converts Microsoft Office docx documents to equivalent text documents"
+  homepage "http://docx2txt.sourceforge.net/"
+  url "http://downloads.sourceforge.net/docx2txt/docx2txt-1.4.tgz"
+  sha256 "b297752910a404c1435e703d5aedb4571222bd759fa316c86ad8c8bbe58c6d1b"
+
+  depends_on "unzip"
+
+  resource "sample_doc" do
+    url "http://calibre-ebook.com/downloads/demos/demo.docx"
+    sha256 "269329fc7ae54b3f289b3ac52efde387edc2e566ef9a48d637e841022c7e0eab"
+  end
+
+  def install
+    system "make", "install", "CONFIGDIR=#{etc}"
```

Comment:
I suspect your errors in running `brew test` here stem from the fact you've not set a prefix here, so it'll be installing into the default prefix rather than the Cellar as intended.